data/pigui/modules/station-view/04-shipMarket.lua

```diff
@@ -281,17 +286,14 @@ function FormatAndCompareShips:draw_atmos_pressure_limit_cell(desc)
 	self:compare_and_draw_column( desc, atmoSlot, self.def.atmosphericPressureLimit, self.b.def.atmosphericPressureLimit )
 end
 
-function FormatAndCompareShips:new(def,b)
-	o = {}
-	setmetatable( o, self )
-	self.__index = self
-	o.column = 0
-	o.emptyMass = def.hullMass + def.fuelTankMass
-	o.fullMass = def.hullMass + def.capacity + def.fuelTankMass
-	o.massAtCapacity = def.hullMass + def.capacity
-	o.cargoCapacity = def.equipSlotCapacity["cargo"]
-	o.def = def
-	o.b = b
+function FormatAndCompareShips:Constructor(def, b)
+	self.column = 0
+	self.emptyMass = def.hullMass + def.fuelTankMass
+	self.fullMass = def.hullMass + def.capacity + def.fuelTankMass
+	self.massAtCapacity = def.hullMass + def.capacity
+	self.cargoCapacity = def.equipSlotCapacity["cargo"]
+	self.def = def
+	self.b = b
 	return o
```

Comment:
`return o` here has no effect - the new instance (`self` in this function) will be returned from `FormatAndCompareShips.New()`.

Reply:
Thanks